compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1249,6 +1256,9 @@ getPackageIdIfFresh logger path BuildMultiPackageConfig {..} sourceDepPids = do
       logDebug $ "Source files are " <> (if sourceFilesCorrect then "not " else "") <> "stale."
       pure $ if sourceDepsCorrect && sourceFilesCorrect then getArchiveDalfPid bmName bmVersion else Nothing
 
+damlMultiBuildVersion :: UnresolvedReleaseVersion
+damlMultiBuildVersion = either throw id $ parseUnresolvedVersion "2.8.0-snapshot.20231018.0"
```

Comment:
Will this string be updated at each release?

Reply:
This is the minimum version where Daml multibuild is enabled by default - it shouldn't have to be changed since it's only used as a comparison point.